Backend/src/main/java/ca/mcgill/cooperator/controller/CourseOfferingController.java

```diff
@@ -37,6 +40,29 @@ public CourseOfferingDto getCourseOfferingById(@PathVariable int id) {
                 courseOfferingService.getAllCourseOfferings());
     }
 
+    @GetMapping("/years")
+    public List<String> getCourseOfferingYears() {
+        List<String> years = new ArrayList<>();
```

Comment:
Use a Set instead, then you don't need to check if the list already contains a year